index.html

```diff
@@ -5,62 +5,160 @@
 
-    </section>
+    <div class="margin absolute flex">
+
+        <iframe src="https://www.youtube.com/embed/V74l_zS1x8E" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+
+        <section class="recomended-videos">
+
+            <div class="flex videos-spacing">
+                <figure><img src="./Images/thumbnail1.png" alt="TOP 5 GAMING MICE 2019"></figure>
+
+                <div>
+                    <p>TOP 5 GAMING MICE 2019</p>
+                    <h6>asdasd<br>1,000 000 Views</h6>
+                </div>
+            </div>
+            <div class="flex videos-spacing">
+                <figure><img src="./Images/thumbnail2.png" alt="Covid-19 animation: how it destroys you.."></figure>
+
+                <div>
+                    <p>Try to watch this without dying</p>
+                    <h6>balerum <br>1,000 000 Views</h6>
+                </div>
+
+            </div>
+            <div class="flex videos-spacing">
+                <figure><img src="./Images/thumbnail3.png" alt="Try to watch this without dying"></figure>
+
+
+                <div>
+                    <p>Covid-19 animation: how it destroys you..</p>
+                    <h6>balerum <br>1,000 000 Views</h6>
+                </div>
+
+            </div>
+            <div class="flex videos-spacing">
+
+                <figure> <img src="./Images/thumbnail4.png" alt="Pork belly combo chicharrón"></figure>
+
+                <div>
+                    <p>Pork belly combo chicharrón</p>
+                    <h6>balerum <br>1,000 000 Views</h6>
+                </div>
+
+            </div>
+            <div class="flex videos-spacing">
+
+                <figure><img src="./Images/thumbnail5.png" alt="Porch pirate vs. glitter bomb"></figure>
+
+                <div>
+                    <p>Porch pirate vs. glitter bomb</p>
+                    <h6>balerum <br>1,000 000 Views</h6>
+                </div>
+            </div>
+            <div class="flex videos-spacing">
+
+                <figure><img src="./Images/thumbnail6.png" alt="Porch pirate vs. glitter bomb"></figure>
+                <div>
+                    <p>Playing flappy bird, every like makes it faster (world record..)</p>
+                    <h6>balerum <br>1,000 000 Views</h6>
+                </div>
+            </div>
+            <div class="flex videos-spacing">
+
+                <figure><img src="./Images/thumbnail7.png" alt="
+                    Eat in one bite challenge | Tacos, Burritos (People Vs)"></figure>
+                <div>
+                    <p>Eat in one bite challenge | Tacos, Burritos (People Vs)</p>
+                    <h6>balerum <br>1,000 000 Views</h6>
+                </div>
+            </div>
+        </section>
+
+    </div>
```

Comment:
These images are not properly loading to the page. Kindly check and fix the errors in here.

Here is a screenshot from your webpage.

![image](https://user-images.githubusercontent.com/57414789/81355166-2e52ec00-9093-11ea-8100-449d35fea27c.png)

Reply:
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/61803988/81373913-7a1d8980-90c3-11ea-88b1-44ea88a3a343.gif)

made the required changes!